packages/foreman/foreman/foreman.spec

```diff
@@ -22,6 +22,9 @@ Source4: %{name}.cron.d
 Source5: %{name}.tmpfiles
 BuildArch:  noarch
 
+# Plugin was removed in Foreman 3.3, 3.5 includes DB cleanup
+Obsoletes: rubygem-foreman_docker < 5.0.0-4
```

Comment:
I perused our EL8 plugin repositories and I could not find a Foreman release where foreman_docker was packaged in the plugins repository.

Reply:
http://koji.katello.org/koji/buildinfo?buildID=57188 says it was there between 2.1 and 2.4?